### DISCUSSION TITLE
Build error since new release

### DISCUSSION THREAD

**Discussion by Ramzza** [pmndrs/zustand#2621] [Bug report]
### Bug Description

Hello!
React material-table is using the "zustand-latest" as a dependency. 
Since this morning we are getting an npm build error:

ERROR in ./node_modules/zustand/esm/index.js 1:0
13:29:16  Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (1:0)
13:29:16  You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
13:29:16  > import { createStore } from 'zustand/vanilla';
13:29:16  | export * from 'zustand/vanilla';
13:29:16  | import ReactExports from 'react';
13:29:16   @ ./node_modules/@material-table/core/dist/store/LocalizationStore.js 12:15-33
13:29:16   @ ./node_modules/@material-table/core/dist/index.js 116:25-61 

### Reproduction Link

Cannot provide example

**Comment by dai-shi** [comment]
Thanks for reporting.
Can you identify if it's the same issue as https://github.com/pmndrs/jotai/discussions/2579?

**Reply by Ramzza** [reply]
Seems quite similar, but I cannot confirm it 100% as I am not so familiar with next.js.

**Comment by dai-shi** [comment]
> React material-table is using the "zustand-latest" as a dependency.

That sounds serious...

**Comment by dai-shi** [comment]
> ERROR in ./node_modules/zustand/esm/index.js 1:0

Ohhh, this is bad. I see why they can complain this.

**Comment by dai-shi** [comment]
https://github.com/pmndrs/zustand/pull/2622 should be the fix.

I wonder if anyone can try and confirm it.

https://ci.codesandbox.io/status/pmndrs/zustand/pr/2622
☝️ Find "Local Install Instructions"

**Reply by zburke** [reply]
We faced the same error in our builds; #2622 fixed it for us.

**Reply by dai-shi** [reply]
Thanks!

**Reply by dai-shi** [reply]
Published: https://www.npmjs.com/package/zustand/v/4.5.4
